crates/swc_common/src/syntax_pos/hygiene.rs

```diff
@@ -138,85 +133,6 @@ impl Mark {
         Mark(raw)
     }
 
-    #[inline]
-    pub fn parent(self) -> Mark {
```

Comment:
I search through the codebase, and find this function is never used, and also `MarkData.parent` field is never used too, so delete it.

Reply:
This should not be deleted

Reply:
And this